docs/fields/file.md

```diff
@@ -32,5 +32,5 @@ field.
         attachment:
             type: file
             extensions: [ png, pdf ]
-            upload: portfolio
+        upload: portfolio
 ```
```

Comment:
Just a note, the original indentation was correct, it is a [field property](https://docs.bolt.cm/3.2/cheatsheet#field-options-for-contenttypes-in-contenttypes-yml) rather than a ContentType property, but I'll grab it all on merge :+1: